packages/scripts/src/commands/app-data/postProcess/assets.ts

```diff
@@ -95,16 +105,30 @@ export class AssetsPostProcessor {
     assetEntries: IAssetEntryHashmap,
     missingEntries: IAssetEntryHashmap
   ) {
-    const contentsTarget = path.resolve(appAssetsFolder, "contents.json");
-    fs.writeFileSync(contentsTarget, JSON.stringify(sortJsonKeys(assetEntries), null, 2));
-    const missingTarget = path.resolve(appAssetsFolder, "untracked-assets.json");
-    if (fs.existsSync(missingTarget)) fs.removeSync(missingTarget);
-    if (Object.keys(missingEntries).length > 0) {
-      logWarning({
-        msg1: "Assets override found without corresponding entry",
-        msg2: Object.keys(missingEntries).join("\n"),
-      });
-      fs.writeFileSync(missingTarget, JSON.stringify(sortJsonKeys(missingEntries), null, 2));
+    try {
```

Comment:
I think better than a `try/catch` block might be to handle the specific case of the assets folder not existing (i.e. the case where there are no assets).

Instead of `try/catch`, you could use an `if` statement along the lines of:
```
if(fs.existsSync(appAssetsFolder)) {
...all the logic in the 'try' block
} else {
...log our error
}
```